docs/faq.md

```diff
@@ -129,3 +129,22 @@ Run "docker system df" to see how much space can be reclaimed. If the space is l
 - (3) Check /tmp directory. 
 
 Run command "du -h / | awk '$1~/[0-9]*G/{print $0}'" to list all the directory which consumes more than 1G spaces. Clean the data not used any more. 
+
+### Q: How to change Docker install path for PAI?
+
+A: If we install Docker on a small disk, when under disk pressure, the cleaner may activate and kill jobs to prevent system crash. We'd better to move Docker to a larger disk. We can change Docker install path through following steps:
+
+1.	Change the Docker path on PAI node.  
+2.	Stop ALL service of the cluster.  
+```sudo ./paictl.py service stop```
+3.	Destroy k8s cluster.  
+  ```sudo ./paictl.py cluster k8s-clean –p /path/to/config```
+4.  Modify your config file layout.yaml. Change "docker-data" to your new Docker path. See sample [layout.yaml](../blob/master/examples/cluster-configuration/layout.yaml#L55)  
+5.	Restart k8s cluster with updated config.  
+  ```sudo ./paictl.py cluster k8s-bootup –p /path/to/new/config```
```

Comment:
Why is it needed? Is this configured in OpenPAI configuration or docker configuration?

Reply:
It's OpenPAI ocnfiguration

Reply:
I prefer separate them. I will mention  step 3 configured path in step 4.

Reply:
I mean create an issue to simplify it, so that user can update configuration easier. And we have a chance to clean up old folder, if it's changed.

Reply:
I think this is an other topic and we need to discuss it later.
How to setup docker cache path is admin behavior. I think we'd better not touch it, just tell them when they change docker cache path, the OpenPAI config should be changed, too.
Any other suggestions about the doc?